tests/integration/clientv3/lease/lease_test.go

```diff
@@ -189,15 +189,15 @@ func TestLeaseKeepAlive(t *testing.T) {
 	}
 }
 
-func TestLeaseKeepAliveOneSecond(t *testing.T) {
+func TestLeaseKeepAliveSeconds(t *testing.T) {
 	integration2.BeforeTest(t)
 
 	clus := integration2.NewCluster(t, &integration2.ClusterConfig{Size: 1})
 	defer clus.Terminate(t)
 
 	cli := clus.Client(0)
 
-	resp, err := cli.Grant(context.Background(), 1)
```

Comment:
one second is not long enough to allow the following statement - `cli.KeepAlive` to have chance to file request, especially when CI env is unstable or busy. it makes sense to increase ttl to 3 seconds.